build/NugetConfigFile.targets

```diff
@@ -0,0 +1,58 @@
+<Project ToolsVersion="15.0">
+  <Target Name="WriteNugetConfigFile">
+
+    <PropertyGroup>
+      <NugetConfigHeader>
+        &lt;?xml version=&quot;1.0&quot; encoding=&quot;utf-8&quot;?&gt;
+        &lt;configuration&gt;
+        &lt;packageSources&gt;
+        &lt;!--To inherit the global NuGet package sources remove the &lt;clear/&gt; line below --&gt;
+        &lt;clear /&gt;
+      </NugetConfigHeader>
+
+      <NugetConfigFeedsPrivate>
+        &lt;add key=&quot;PrivateBlobFeed&quot; value=&quot;$(ExternalRestoreSources)&quot; /&gt;
+      </NugetConfigFeedsPrivate>
+      
+      <NugetConfigFeedsNormal>
+        &lt;add key=&quot;BlobFeed&quot; value=&quot;https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json&quot; /&gt;
```

Comment:
You can probably make this prettier using `CDATA`, so you don't have to escape all the brackets.  See an example here: https://stackoverflow.com/a/2784200